easybuild/easyconfigs/p/ParMETIS/ParMETIS-4.0.3-intel-2015b.eb

```diff
@@ -0,0 +1,22 @@
+name = 'ParMETIS'
+version = '4.0.3'
+
+homepage = 'http://glaros.dtc.umn.edu/gkhome/metis/parmetis/overview'
+description = """ParMETIS is an MPI-based parallel library that implements a variety of algorithms for partitioning unstructured graphs,
+ meshes, and for computing fill-reducing orderings of sparse matrices. ParMETIS extends the functionality provided by METIS and includes
+ routines that are especially suited for parallel AMR computations and large scale numerical simulations. The algorithms implemented in
+ ParMETIS are based on the parallel multilevel k-way graph-partitioning, adaptive repartitioning, and parallel multi-constrained partitioning
+ schemes."""
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'optarch': True, 'usempi': True, 'pic': True}
+
+source_urls = [
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/parmetis',
+    'http://glaros.dtc.umn.edu/gkhome/fetch/sw/parmetis/OLD',
+]
+sources = [SOURCELOWER_TAR_GZ]
+
+builddependencies = [('CMake', '3.1.0', '-GCC-4.9.2', True)]
```

Comment:
please replace with:

```python
builddependencies = [('CMake', '3.1.0', '-GCC-4.9.2', True)]
```

Your current entry means: CMake v3.1.0 with versionsuffix `-GCC-4.9.2` and dummy toolchain.

This doesn't really matter with the standard module naming scheme, but is incorrect in the context of a custom module naming scheme like `HierarhicalMNS` and will cause the corresponding module not to be found.